prefect_databricks/flows.py

```diff
@@ -249,6 +249,14 @@ def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_paramete
             return multi_task_runs
         ```
     """  # noqa
+    warnings.warn(
+        "The flow `jobs_runs_submit_and_wait_for_completion` will be moved to"
+        " prefect_databricks.jobs.flows in the next release, but can be imported with"
+        " `from prefect_databricks.jobs import"
+        " jobs_runs_submit_and_wait_for_completion`",
```

Comment:
I think this is a little more explicit.
```suggestion
        "jobs_runs_submit_and_wait_for_completion has been moved to the "
        "prefect_databricks.jobs module. jobs_runs_submit_and_wait_for_completion "
        "is available from prefect_databricks.flows for backwards compatibility, but will "
        "be removed in a subsequent release. Please update your code to import ",
        "jobs_runs_submit_and_wait_for_completion from prefect_databricks.jobs."
```